packages/create-atomic/templates/public/index.hbs

```diff
@@ -0,0 +1,69 @@
+<html lang='en'>
```

Comment:
If I remember correctly from your design doc, down the line, these files (index.hbs, layout.css, theme.css) would be { returned/powered/controlled/templated/etc } by the search interface service as the source of truth, right ?

Reply:
Yes for phase 2 https://docs.google.com/document/d/1QzqezewjxUNdd2MhEQ4bmMT_0K1PcQbF3FBhzR2yka4/edit#heading=h.xc32y4ockguh

Currently there is no endpoint that returns the Atomic html configuration